### DISCUSSION TITLE
Mehrere Buttons nebeneinander

### DISCUSSION THREAD

**Discussion by hopedotz** [Zefau/ioBroker.jarvis#868] [Q&A]
Servus! Ist es möglich, mehrere Buttons **nebeneinander** anzeigen zu lassen? Bisher gehts ja nur untereinander. Oder ich übersehe etwas...

**Comment by Zefau** [comment]
Aktuell nicht. Was genau stellst du dir vor? Wie ist der Anwendungsfall?

**Comment by hopedotz** [comment]
Ich hätte z.B. gerne mehrere Buttons nebeninander, damit ich verschiedene Lichtszenen mit einem Klick steuern kann.

**Comment by Zefau** [comment]
Geht leider nicht aktuell. Leg gerne ein Feature Request an.
